docs/source/tutorials/EasyCV 度量学习resnet50.ipynb

```diff
@@ -0,0 +1,333 @@
+    "import torch\n",
+    "import os\n",
+    "os.environ['CUDA']='cu' + torch.version.cuda.replace('.', '')\n",
+    "os.environ['Torch']='torch'+torch.version.__version__.replace('+PAI', '')\n",
+    "!echo $CUDA\n",
+    "! echo $Torch"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "99e24dec-b747-4a23-a981-9aae14e433cf",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "# install some python deps\n",
+    "! pip install mmcv-full==1.4.4 -f https://download.openmmlab.com/mmcv/dist/${CUDA}/${Torch}/index.html\n",
+    "! pip install http://pai-vision-data-hz.oss-cn-zhangjiakou.aliyuncs.com/third_party/nvidia_dali_cuda100-0.25.0-1535750-py3-none-manylinux2014_x86_64.whl"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6166dd4e-0e31-4061-9229-5f79968732e0",
+   "metadata": {
+    "jp-MarkdownHeadingCollapsed": true,
+    "tags": []
+   },
+   "source": [
+    "3.  安装EasyCV算法包"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d55d2b68-b9d3-484a-abf6-561187153c54",
+   "metadata": {
+    "jp-MarkdownHeadingCollapsed": true,
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "pip install pai-easycv"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d5e4d314-f59f-464c-950d-12c94b11020c",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "## 简单验证"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "80c51674-b1cc-4acf-a8e3-ed20306afebf",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from easycv.apis import *"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0512498b-a900-4969-9563-67029a1009e2",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "## 运行环境要求\n",
+    "\n",
+    "PAI-Pytorch 1.7/1.8镜像， GPU机型 P100 or V100， 内存 32G"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8df2d210-4bcf-4b41-a981-b264c41e91a7",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "## CUB200 度量学习"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "fce3f492-cfe0-4121-8f7d-877b31e3438b",
+   "metadata": {},
+   "source": [
+    "下面示例介绍如何利用[CUB200](http://www.vision.caltech.edu/datasets/cub_200_2011/)数据，使用ResNet50模型快速进行度量学习模型的训练评估、模型预测过程"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6e769ad6-3ae0-4d5b-bf23-55673a239aa8",
+   "metadata": {},
+   "source": [
+    "### 数据准备\n",
+    "下载cub200数据，解压到`data/cub200`目录， 目录结构如下\n",
+    "\n",
+    "```text\n",
+    "data/cub200\n",
+    "├── images\n",
+    "├── images.txt\n",
+    "├── image_class_labels.txt\n",
+    "├── train_test_split.txt\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "62eeff3d-274d-4ff9-8123-8e11d97c0d38",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "! mkdir -p data/cub200 && wget https://data.caltech.edu/tindfiles/serve/1239ea37-e132-42ee-8c09-c383bb54e7ff/ &&  tar -zxf CUB_200_2011.tgz -C data/cub200/"
```

Comment:
prepare data package in oss://pai-vision-data-hz/EasyCV/datasets/ and provide public-read url